src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -28,6 +28,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string ImplementInterface = "Implement Interface";
         public const string InsertMissingCast = "InsertMissingCast";
         public const string MoveToTopOfFile = "Move To Top Of File";
+        public const string QualifyMemberAccess = "Qualify member access";
```

Comment:
Confirmed that they're only used in ordering attributes.  I've updated all of them to use `nameof()`.